### DISCUSSION TITLE
OpenDTU friert ein sobald Panel angeschlossen wird

### DISCUSSION THREAD

**Discussion by klingbing** [tbnobody/OpenDTU#1160] [Q&A]
Hallo in die Runde,

zunächst vielen Dank für die Arbeit aller Beteiligten hier!

Ich habe da mal ein Problem, ich gehe unmittelbar nicht von einem Bug (sondern vielmehr dem üblichen Problem VOR dem Bildschirm) aus:

Ich habe bereits vor einigen Tagen erfolgreich eine OpenDTU bestehend aus einem WT32-ETH01 mit [EBYTE E01-ML01DP5](https://de.aliexpress.com/item/1005001802605439.html) aufgesetzt. Als device profiile verwende ich [wt32-eth01.json](https://github.com/tbnobody/OpenDTU/blob/master/docs/DeviceProfiles/wt32-eth01.json) ohne Display.

Soweit so gut, alles funktioniert, ich komme auf die Oberfläche, unter Info -> System -> nRF24 Status, nRF24 Chip-Status und nRF24 Chip-Type alles im grünen Bereich.

Nun sind auch der Wechselrichter (HM-800) und die Panele angekommen, also: Ein Panel an den WR angeschlossen, gewartet bis die grüne LED im 4 Sek. Takt aufblinkt, WR in die Steckdose gesteckt und OpenDTU Stromversorgung hergestellt. Ich komme dann auf die OpenDTU, kann sehen dass die Zeit-Sync nicht stimmt. Sobald ich 1-2 weitere Klicks in der OpenDTU vornehme friert sie mit einem Spinning Wheel of Death ein, die OpenDTU ist dann auch nicht mehr im Netzwerk erreichbar. Wenn ich das Panel abklemme und die OpenDTU power cycle komme ich wieder auf die OpenDTU. 

Wenn ich zuerst die OpenDTU anklemme, dann den WR und dann das eine Panel hört das Log einfach auf, siehe:

`00 00 21 CB DD 
11:01:22.846 > RX Period End
11:01:22.846 > All missing
11:01:22.846 > Nothing received, resend whole request
11:01:22.846 > TX AlarmData Channel: 3 --> 15 90 13 84 32 80 19 68 80 80 11 00 64 B8 F7 DC 00 00 00 00 00 00 00 00 21 CB DD 
11:01:23.563 > RX Period End
11:01:23.563 > All missing
11:01:23.563 > Nothing received, resend count exeeded
11:01:23.614 > TX SystemConfigPara Channel: 23 --> 15 90 13 84 32 80 19 68 80 80 05 00 64 B8 F7 DC 00 00 00 00 00 00 00 00 35 DF C9 
11:01:23.871 > RX Period End
11:01:23.871 > All missing
11:01:23.871 > Nothing received, resend whole request
11:01:23.871 > TX SystemConfigPara Channel: 40 --> 15 90 13 84 32 80 19 68 80 80 05 00 64 B8 F7 DC 00 00 00 00 00 00 00 00 35 DF C9  `

Bisheriges Troubleshooting:
- NTP eingetragen und Verbindung zu diesem geprüft
- Sendeleistung erhöht
- Seriennummer des WR mehrfach abgeglichen

Fragen:
- Ich kann den WR auch mit nur einem Panel betreiben, right? Die DC-Stecker des nicht belegten Anschlusses müssen dabei nicht verbunden werden, right?
- Die Stromversorgung der OpenDTU erfolgt derzeit (noch) über den USB zu TTL Adapter, kann das der Bottleneck sein, auch wenn die OpenDTU problemlos läuft so lange kein Panel angeschlossen ist?
- Und zu guter letzt: Wo kann ich sonst noch ansetzen..?

Im voraus DANKE für eure Hilfe!

**Comment by Omega13x** [comment]
Ich tippe auf die Stromversorgung, wird wohl über den TTL-Adapter zu wenig sein, besonders das NRF-Modul zieht ordentlich Strom im Sendebetrieb. Dann einfach mal beide PV-Module anschließen. Der Aufwand sollte sich in Grenzen halten. Hast du einen Elko (z.B. 100µf) in der Stromversorgung zum NRF installiert? Was ich noch empfehlen kann, ist sich auf Discord "https://discord.gg/WzhxEY62mB" anzumelden. Da wird dir bestimmt geholfen, wenn meine Tipps nicht zum Erfolg führen.

**Reply by klingbing** [reply]
Danke für deine Rückmeldung, da lag ich mit meiner Vermutung (Stromversorgung) womöglich nicht ganz falsch - ich werde das testen und hier feedbacken.

Den Elko habe ich nicht verbaut, ich meine irgendwo aufgeschnappt zu haben, dass dieser bei dem von mir verwendeten NRF-Modul nicht notwendig sei.

**Comment by DPO99** [comment]
Gem. Info werden mindestens 1A benötigt. Usb zu TTL ist doch hart verkabelt oder? Was haste denn für ein Netzteil? Sag aber bitte nicht USB Buchse am PC.... :D 

```
Power supply
Use a power suppy with 5 V and 1 A. The USB cable connected to your PC/Notebook may be powerful enough or may be not.
```

**Reply by klingbing** [reply]
Nix da Buchse am PC! :D Ist schon n ok‘es Netzteil mit 1,2A, hart verkabelt jo. Ich teste ein anderes NT ohne usb zu ttl.

**Reply by Eddy-Fischer** [reply]
Servus Klinging, mein WT32 ETH01 hat aufgehört zu funktionieren, nachdem ich vor 2 Wochen update von github gemacht habe (da habe ich das vergessen zu machen ("If you are upgrading from a version before 15.03.2023 you have to upgrade the partition table of the ESP32. Please follow the [this](https://github.com/tbnobody/OpenDTU/blob/master/docs/UpgradePartition.md) documentation!"). Seitdem funktioniert es nicht mehr. Hast du eine Idee wie ich es reparieren? Ich habe auch versuch nochmal von Anfang zu flashen mit altem Verfahren: 
boot_app0.bin
bootloader.bin
opendtu-generic.bin
partitions.bin

und mit dem neuen Verfahren ein einziege Datei opendtu-generic.factory.bin.
Beide Verfahren hat nicht funktioniert.
Könntest du mir hier dabei helfen? Meine Frau hat Probleme mit Wifi Signale zu Hause. Daher benutze ich Ethernet/LAN.
Danke und Gruss
Ed

**Reply by klingbing** [reply]
Moin Ed, ich bin womöglich noch unerfahrener als du mit OpenDTU und erst seit wenigen Wochen dabei, sodass mich das Upgrade der Partitionstabelle garnicht betraf.

Mach doch am besten einen neuen Thread mit Logfiles auf, ich vermute (!), dass es sich um unterschiedliche Probleme handelt.
